packages/flutter_tools/test/integration.shard/gen_l10n_test.dart

```diff
@@ -157,11 +157,4 @@ void main() {
     final StringBuffer stdout = await runApp();
     expectOutput(stdout);
   });
-
-  test('generated l10n classes produce expected localized strings with deferred loading', () async {
```

Comment:
Add a TODO to add the test back, and (nit) since we do need a rewrite, this is probably ok to just remove, but may also consider using `}, skip: true);`

Reply:
added a TODO